shopfloor_single_product_transfer/services/single_product_transfer.py

```diff
@@ -106,22 +106,21 @@ def _scan_location__check_stock(self, location):
         )
         if not quants_in_location:
             message = self.msg_store.location_empty(location)
-            return self._response_for_select_location(message=message)
+            return self._response_for_select_location_or_package(message=message)
 
     def _scan_location__check_package(self, location):
         """Check if the location has lines without an assigned package."""
         lines_without_package = self.env["stock.move.line"].search(
             [
                 ("location_id", "=", location.id),
                 ("package_id", "=", False),
-                ("picking_id.picking_type_id", "in", self.picking_types.ids),
```

Comment:
I would have kept this `("picking_id.picking_type_id", "in", self.picking_types.ids)`